### PR TITLE
Update runtime to KDE 5.12

### PR DESCRIPTION
**Pull request by FakeShemp**

### CODE REVIEW

File: org.kde.kdiamond.json [hunk 1]
```diff
@@ -10,29 +10,28 @@
         "--socket=x11",
         "--socket=wayland",
         "--socket=pulseaudio",
-        "--device=dri",
-        "--filesystem=xdg-config/kdeglobals:ro"
```
Comment (tsdgeos) [inline org.kde.kdiamond.json]: Why did you remove this?

Comment (FakeShemp) [reply org.kde.kdiamond.json]: As far as I know, it's set by default now in 5.12. Please correct me if I'm wrong though.

Comment (aleixpol) [reply org.kde.kdiamond.json]: flatpak-builder will inherit some properties from the runtime, such as kdeglobals. Nevertheless, `--device=dri` isn't one of these so you probably still want it if you want graphics hardware acceleration.

Comment (FakeShemp) [reply org.kde.kdiamond.json]: `--device=dri` is still there, but I had to remove the comma to not bork the json.
